PerfettoCds/Pipeline/Tables/PerfettoCpuSchedTable.cs

```diff
@@ -114,6 +154,11 @@ public static void BuildTable(ITableBuilder tableBuilder, IDataExtensionRetrieva
                     DurationColumn,
                     EndStateColumn,
                     PriorityColumn,
+                    WakeEventFoundColumn,
```

Comment:
Could we make a separate tableConfig that includes WantEvents? "CPU Scheduling, Wake Events". Because WakeEvents can be null (not sure how common this is), we don't want the user to be forced to have a bunch of empty columns.